pinot-server/src/main/java/com/linkedin/pinot/server/starter/helix/SegmentMessageHandlerFactory.java

```diff
@@ -89,6 +91,11 @@ public String getMessageType() {
     return Message.MessageType.USER_DEFINE_MSG.toString();
   }
 
+  @Override
+  public List<String> getMessageTypes() {
+    return ImmutableList.of(Message.MessageType.USER_DEFINE_MSG.name());
```

Comment:
Can the result change between invocations? If not, why not use static final to avoid new object creation?

Reply:
makes sense.